step.yml

```diff
@@ -31,6 +31,26 @@ toolkit:
     package_name: github.com/bitrise-steplib/steps-sign-apk
 inputs:
   - apk_path: "$BITRISE_APK_PATH"
+    opts:
```

Comment:
pls do it similarly: https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/step.yml#L170-L185

Reply:
also clear the default value, this way the step logic can decide if the input has value because a user defined it or the value is the default.